tests/api_error.test.ts

```diff
@@ -0,0 +1,13 @@
+import { fetchDataFromAPI } from "./utils/helper";
+
+// test used for all error tests
+describe("Test API Errors", () => {
+  it("test /coins/alex route ERROR", async () => {
+    // check to make sure a 403 error is send if the route or authtoken is incorrect
+    await expect(
+      fetchDataFromAPI("http://localhost:8000/coins/alex")
+    ).rejects.toMatchObject({
+      response: { status: 403 },
+    });
+  });
```

Comment:
Will these tests be the same for all endpoints, no matter the language?

Reply:
Yeah all REST APIs should be able to return a 403 status. I was able to do it in go and node and I am 90% sure pretty much all other REST frameworks/languages should be able to return that.